src/connector.rs

```diff
@@ -484,6 +502,8 @@ impl EthConnectorContract {
     /// We starting early checking for message data to avoid `ft_on_transfer` call panics
     /// But we don't check relayer exists. If relayer doesn't exist we simply not mint/burn the amount of the fee
     pub fn ft_transfer_call(&mut self) {
+        //TODO: perhaps need to add pausability functionality here as well?
```

Comment:
How will we decide whether this should be done or not?

Reply:
This is a public NEP-141 method, so we need to think if we want to add pausability for bridged Ether as a separate flag (i.e. `PAUSED_TRANSFER`).
@artob

Reply:
NEP-141 has very specific requirements on what to add, I would just stick with those. I don't think it is a good idea to add plausibility. May lead to unintended consequences and could actually break compatibility.